.gitignore

```diff
@@ -60,8 +54,12 @@ flatpak-build/
 # Lua
 lua-*
 
-# Clangd 
+# Clangd
 compile_commands.json
 .clangd
 .cache
 Debug
+
+# Miscellaneous
+.DS_STORE
```

Comment:
This will only match the top level directory, but you removed the others. Can you elaborate, why the others are not matched anymore?

Reply:
The only files that shouldn't be matched anymore are hypothetical ones that look like `somename.DS_STORE`. In github/gitignore, the [macOS](https://github.com/github/gitignore/blob/main/Global/macOS.gitignore) gitignore file didn't have an asterick, so I opted to remove it from here for simplicity.

For your concern about only matching the top level directory, I tried to address that with the other comment I made, does that answer your question?

Reply:
macOS can create `.DS_STORE` files in any folder, so this should ideally be a wildcard like: `**/.DS_STORE`